content/pandas/concepts/groupby/terms/count/count.md

```diff
@@ -0,0 +1,54 @@
+---
+Title: '.count()'
+Description: 'Produces a result with counts of the values for each group in a GroupBy object.'
```

Comment:
```suggestion
Description: 'Produces a new Series or DataFrame with counts of the values for each group in a GroupBy object.'
```

Reply:
We typically mention the data type in these docs, but for all these methods there is no discussion of the return type, only that a 'result' is returned. I don't think it has to be in the description but I think somewhere it should be stated that the return will be a series or dataframe.